charts/sourcegraph/templates/searcher/searcher.StatefulSet.yaml

```diff
@@ -89,7 +81,7 @@ spec:
           timeoutSeconds: 5
         volumeMounts:
         - mountPath: /mnt/cache
-          name: cache-ssd
+          name: cache
         - mountPath: /tmp
```

Comment:
What's this tmp dir for?

Reply:
Not sure what it's used for, but it was in the deployment originally, so I kept it

Reply:
It was added [here](https://github.com/sourcegraph/deploy-sourcegraph-helm/commit/772bf4739649df7323a358e201416ef685f74098) it looks like a while back. If we know it isn't used, I'm fine to delete, but wanted to keep consistent from what was created in the past.